sp800-63a/sec5_proofing.md

```diff
@@ -193,27 +193,23 @@ The following requirements apply to the identity verification steps for IAL 2 an
 
 
 
-- The CSP SHOULD verify knowledge of recent transactional history that the CSP is a participant to.  The CSP SHALL ensure transaction information meets the minimum entropy for a Memorized Secret. For example, verification of amount and confirmation number of a micro-deposit to a claimed and valid bank account.
-
-
-
-- The CSP MAY perform KBV by asking questions of the claimed identity to demonstrate they are the owner of the claimed information. 
+- The CSP SHOULD perform KBV by verifying knowledge of recent transactional history that the CSP is a participant to.  The CSP SHALL ensure transaction information meets the minimum entropy for a Memorized Secret. For example, verification of amount and confirmation number of a micro-deposit to a claimed and valid bank account. The CSP MAY perform KBV by asking questions of the claimed identity to demonstrate they are the owner of the claimed information. However, the following requirements apply:
```

Comment:
"participant to" -> "participant in"
"ensure" -> "ensure that"
Memorized secrets don't have any explicit entropy requirements. However, in most other places, we require 20 bits of entropy (OTP authenticator) outputs, for example) so maybe just say "20 bits of entropy" here.
The sentence starting "For example," needs a verb.
"questions of the claimed identity" -> "questions of the claimant"